Scripts/script-FindSimilarIncidents.yml

```diff
@@ -96,50 +95,34 @@ script: |-
           if isinstance(label_value, list):
               label_value.sort()
               labels_map[label_key] = label_value
-      if filter_labels:
-          labels_map = {k: v for k, v in labels_map.items() if k in filter_labels}
-      return labels_map
-
-
-  def get_context_values(context_keys):
-      context_values = {}
-      missing_keys = []
-      incident_context = nested_dict_flatted(demisto.context())
-
-      if incident_context:
-          for key in context_keys:
-              if key:
-                  value = incident_context.get(key)
-                  if value:
-                      context_values[key] = value
-                  else:
-                      missing_keys.append(key)
-      return context_values, missing_keys
 
+      return labels_map
 
   def get_incidents_by_keys(similar_incident_keys, incident_time, incident_id, hours_back, ignore_closed, max_number_of_results):
-      similar_keys_query = " or ".join(map(lambda t: '%s:%s' % (t[0], t[1]), similar_incident_keys.items()))
+      similar_keys_query = " and ".join(map(lambda t: '%s:"%s"' % (t[0], t[1]), similar_incident_keys.items()))
 
       incident_time = parse_datetime(incident_time)
       max_date = incident_time
       min_date = incident_time - timedelta(hours=hours_back)
-      hours_back_query = 'occurred:>="%s" and occurred:<="%s"' % (min_date.isoformat(), max_date.isoformat())
+      hours_back_query = '{0}:>="{1}" and {0}:<="{2}"'.format(TIME_FIELD, min_date.isoformat(), max_date.isoformat())
```

Comment:
so now you can support both   occurred and created time?

Reply:
yes